src/main/java/de/craften/ui/swingmaterial/MaterialTextField.java

```diff
@@ -206,12 +272,12 @@ void update() {
             }
             Color targetColor;
             if (target.getText().isEmpty() && target.isFocusOwner()) {
-                targetColor = MaterialColor.CYAN_500;
+                targetColor = accentColor;
             } else {
                 if (target.getText().isEmpty()) {
-                    targetColor = MaterialColor.MIN_BLACK;
+                    targetColor = new Color(target.getForeground().getRGB() & 0x00FFFFFF | HINT_OPACITY_MASK, true);//MaterialColor.MIN_BLACK;
                 } else {
-                    targetColor = MaterialColor.LIGHT_BLACK;
+                    targetColor = accentColor; //MaterialColor.LIGHT_BLACK;
```

Comment:
We should take a look into the Material Design specs, and I think that the accent color doesn't remain if the component loses focus there.